docs/scripts/object_orientated_programming_poc/oop_poc_activity.py

```diff
@@ -0,0 +1,146 @@
+"""The purpose of this script is to demonstrate how to use the OOP interface for Activity.
+The following actions are shown in this script:
+1. Creating a file with an Activity
+2. Retrieve an activity by parent File
+3. Creating a second file with the same activity
+4. Modifying the activity attached to both Files
+5. Creating a table with an Activity
+"""
+import asyncio
+import os
+
+from synapseclient.models import (
+    File,
+    Activity,
+    UsedURL,
+    UsedEntity,
+    Table,
+    Column,
+    ColumnType,
+)
+import synapseclient
+
+PROJECT_ID = "syn52948289"
+
+syn = synapseclient.Synapse(debug=True)
+syn.login()
+
+
+def create_random_file(
+    path: str,
+) -> None:
+    """Create a random file with random data.
+
+    Arguments:
+        path: The path to create the file at.
+    """
+    with open(path, "wb") as f:
+        f.write(os.urandom(1))
+
+
+async def store_activity_on_file():
+    name_of_file = "my_file_with_random_data.txt"
+    path_to_file = os.path.join(os.path.expanduser("~/temp"), name_of_file)
+    create_random_file(path_to_file)
+
+    name_of_second_file = "my_second_file_with_random_data.txt"
+    path_to_second_file = os.path.join(
+        os.path.expanduser("~/temp"), name_of_second_file
+    )
+    create_random_file(path_to_second_file)
+
+    # Create an activity =================================================================
+    activity = Activity(
```

Comment:
Does the replace the old `Activity` object or do we have two `Activity` objects now?

Reply:
For now we have 2 Activity objects. I intend to deprecate all of the current models (Entity, File, Folder, Activity ect....) when we have everything ready.